web/changelog.json

```diff
@@ -1,5 +1,12 @@
 [
   {
+    "version": "3.8.x",
```

Comment:
I figure it's nice to have a placeholder for people to add things to without necessarily needing to know what the ultimate release number will be.